modules/button/react/lib/types.ts

```diff
@@ -44,6 +44,7 @@ export enum IconButtonVariant {
   CircleFilled = 'circleFilled',
   Inverse = 'inverse',
   InverseFilled = 'inverseFilled',
+  Alternate = 'alternate',
```

Comment:
Maybe `SquareAlternate` ?

Reply:
Naming things is part art, part best practices, and really hard _sometimes_. With button variants, we're at capacity for intuitive names. So disclaimer: I'm not sure I have a good suggestion right now.

My opinion on things to think about when naming things like props, enums, etc., would be curious to hear what others have to say or add:
- **The name should attempt to stand alone. Or be clear on what it relates to.** When using `Alternate` or `SquareAlternate`, it just sounds like you're saying `Other`.
- **It should try to accommodate future changes if/when they happen.** The word `Alternate` is sort of a catch-all. It doesn't allow us to have any expansion in our variants in the future.
- **It should be descriptive without locking you into a particular implementation.** `Alternate` doesn't lock us into a particular implementation (like if `Primary` was named `Blue` instead) but it isn't particularly descriptive. It's almost too vague.

Reply:
Is `Toolbar` too specific? It satisfies #*1* in Lynn's comment above. As far as #*2* and #*3*, can anyone think of another context where a button like this would be used outside of a Toolbar (or anything which could reasonably be called a Toolbar)?

Another suggestion would be `SquareCompact`, but I wonder if it should even use the shared `Square` prefix given all the visual differences in states between this variant and all the current `Square*` variants -- in addition to the differences in the Toggled states, it looks like Focus and Click are also different on their own.

Reply:
i liked the suggestion of `compact` as far as I'm aware a lot of products in workday would be using this as a sub set to a toolbar, it's distinct enough and it avoids confusion with the other variants, another option was `groupedSquare`